python/pyarrow/_acero.pyx

```diff
@@ -553,7 +553,7 @@ cdef class Declaration(_Weakrefable):
         return frombytes(GetResultValue(DeclarationToString(self.decl)))
 
     def __repr__(self):
-        return "<pyarrow.acero.Declaration>\n{0}".format(str(self))
+        return f"<pyarrow.acero.Declaration>\n{str(self)}"
```

Comment:
`str` is implicit in f-strings, you don't need to call it explicitly. Example:
```python
>>> from decimal import Decimal
>>> d = Decimal('1.500')
>>> repr(d)
"Decimal('1.500')"
>>> str(d)
'1.500'
>>> f"d is {d}"
'd is 1.500'
>>> f"d is {d!r}"
"d is Decimal('1.500')"
```

Reply:
Thanks for review! I will change this.

Reply:
Solved in [cef7a3b](https://github.com/apache/arrow/pull/45629/commits/cef7a3bd7cce6d5f2cdf28a905700a787e72e02f)